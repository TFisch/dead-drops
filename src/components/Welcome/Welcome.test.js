import React from 'react';
import { Welcome } from './';
import { shallow } from 'enzyme';

describe('Welcome', () => {
  let wrapper;

  beforeEach(() => {

    wrapper = shallow(<Welcome
    />);
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

});