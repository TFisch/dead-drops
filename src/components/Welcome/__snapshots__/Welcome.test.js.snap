// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1
          className="welcome-text"
>
          WELCOME
</h1>,
        <div
          className="login"
>
          <a
                    className="auth-link"
                    href="https://api.pinterest.com/oauth/?response_type=code&redirect_uri=https://ce2f9bf7.ngrok.io/HandleUser&scope=read_public,write_public,read_relationships,write_relationships&client_id=4987807426915878592"
          >
                    CLICK HERE TO SIGN IN WITH PINTEREST
          </a>
</div>,
      ],
      "className": "welcome",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "WELCOME",
          "className": "welcome-text",
        },
        "ref": null,
        "rendered": "WELCOME",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <a
            className="auth-link"
            href="https://api.pinterest.com/oauth/?response_type=code&redirect_uri=https://ce2f9bf7.ngrok.io/HandleUser&scope=read_public,write_public,read_relationships,write_relationships&client_id=4987807426915878592"
>
            CLICK HERE TO SIGN IN WITH PINTEREST
</a>,
          "className": "login",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "CLICK HERE TO SIGN IN WITH PINTEREST",
            "className": "auth-link",
            "href": "https://api.pinterest.com/oauth/?response_type=code&redirect_uri=https://ce2f9bf7.ngrok.io/HandleUser&scope=read_public,write_public,read_relationships,write_relationships&client_id=4987807426915878592",
          },
          "ref": null,
          "rendered": "CLICK HERE TO SIGN IN WITH PINTEREST",
          "type": "a",
        },
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1
            className="welcome-text"
>
            WELCOME
</h1>,
          <div
            className="login"
>
            <a
                        className="auth-link"
                        href="https://api.pinterest.com/oauth/?response_type=code&redirect_uri=https://ce2f9bf7.ngrok.io/HandleUser&scope=read_public,write_public,read_relationships,write_relationships&client_id=4987807426915878592"
            >
                        CLICK HERE TO SIGN IN WITH PINTEREST
            </a>
</div>,
        ],
        "className": "welcome",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "WELCOME",
            "className": "welcome-text",
          },
          "ref": null,
          "rendered": "WELCOME",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <a
              className="auth-link"
              href="https://api.pinterest.com/oauth/?response_type=code&redirect_uri=https://ce2f9bf7.ngrok.io/HandleUser&scope=read_public,write_public,read_relationships,write_relationships&client_id=4987807426915878592"
>
              CLICK HERE TO SIGN IN WITH PINTEREST
</a>,
            "className": "login",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "CLICK HERE TO SIGN IN WITH PINTEREST",
              "className": "auth-link",
              "href": "https://api.pinterest.com/oauth/?response_type=code&redirect_uri=https://ce2f9bf7.ngrok.io/HandleUser&scope=read_public,write_public,read_relationships,write_relationships&client_id=4987807426915878592",
            },
            "ref": null,
            "rendered": "CLICK HERE TO SIGN IN WITH PINTEREST",
            "type": "a",
          },
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "disableLifecycleMethods": true,
    "hydrateIn": undefined,
  },
}
`;
